.github/workflows/cuda.yml

```
@@ -68,10 +68,15 @@ jobs:
             -DAMReX_ENABLE_TESTS=ON                      \
             -DAMReX_PARTICLES=ON                         \
             -DAMReX_FORTRAN=OFF                          \
-            -DAMReX_OMP=ON                               \
+            -DAMReX_GPU_BACKEND=CUDA                     \
             -DCMAKE_C_COMPILER=$(which nvc)              \
             -DCMAKE_CXX_COMPILER=$(which nvc++)          \
-            -DCMAKE_Fortran_COMPILER=$(which nvfortran)
+            -DCMAKE_CUDA_HOST_COMPILER=$(which nvc++)    \
+            -DCMAKE_Fortran_COMPILER=$(which nvfortran)  \
+            -DCMAKE_Fortran_COMPILER=$(which nvfortran)  \
```

We can probably remove both because Fortran is off.  But at least remove one.

---

Now I am curious to turn it ON :D
```suggestion
            -DCMAKE_Fortran_COMPILER=$(which nvfortran)  \
```